govuk-one-login/authentication-tech-docs
# AP-182 Document the /logout endpoint in the flow diagram

[AP-182](https://govukverify.atlassian.net/browse/AP-182)

## What
Add the `/logout` endpoint to the how it works diagram with explanation

- add /logout endpoint to the diagram to show the role of logout in the overall flow
- enhance the text explaining the new steps in the diagram
- normalise the use of links in the explanatory text to link to appropriate content
- add additional links to the _links.erb partial
- update _changelog.erb
- ignore draw.io temporary workfiles from the git repo

![image](https://github.com/govuk-one-login/authentication-tech-docs/assets/34273121/f33bc8af-a55d-45b9-9a48-bb17232b70f9)


## Why
The use of /logout is part of the overall flow of interaction with the One Login API, we need to ensure logout is seen as part of the overall journey and its omission from the diagram was inconsistent.

## Its done when
- the digram shows the interaction with the logout endpoint
- the explanation reflects this change
- there is an appropriate level of detail
- there is a mention in the changelog

## If we don't do this
- there will still be a gap in one of our key communication assets
- logout will not be presented alongside the other endpoints

## Out of scope
- backchannel logout, which is a separate server to server flow

## Technical writer support

Yes

## How to review

- run the branch locally using the docker preview
- review the diagram in context
- ensure diagram and documentation make sense and are consistent
- check the changelog makes sense

## Changelog

If this change is significant (for example, launching a new feature or deprecating a feature), you should update the changelog found at `partials/_changelog.erb`  under the heading 'Documentation updates'.

## Confirm



[AP-182]: https://govukverify.atlassian.net/browse/AP-182?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
